config/certmanager/certificate.yaml

```diff
@@ -1,22 +1,22 @@
 # The following manifests contain a self-signed issuer CR and a certificate CR.
```

Comment:
Any reason to keep the certmanager directory here, since it is an external requirement (either capi or a pre-req of capi)?

Reply:
These are the certs for the controller as I understood, which are built when running kustomize